src/engraving/rendering/score/horizontalspacing.cpp

```diff
@@ -1599,6 +1603,12 @@ KerningType HorizontalSpacing::doComputeKerningType(const EngravingItem* item1,
         return computeStemSlashKerningType(toStemSlash(item1), item2);
     case ElementType::PARENTHESIS:
         return item2->isBarLine() ? KerningType::NON_KERNING : KerningType::KERNING;
+    case ElementType::ARTICULATION:

```

Comment:
It looks like this code is never reached - if `type1` is an articulation or fermata `computeArticulationAndFermataKerning(item1, item2);` will have taken care of it before this function is called.

Reply:
Urgh thanks, well spotted. A leftover from a previous implementation I tried out